entrypoint.sh

```diff
@@ -4,4 +4,5 @@ STORE=$(realpath /mddb/"${STORE:-cs.repo}")
 MDDB="/mddb/${MDDB:-metadata.db}"
 RECIPES="${RECIPES:-/recipes/}"
 
-/usr/local/bin/bdcs-api-server --bdcs "$STORE" "$MDDB" "$RECIPES"
+mkdir -p /run/weldr
+/usr/local/bin/bdcs-api-server -s /run/weldr/api.socket -g root --bdcs "$STORE" "$MDDB" "$RECIPES"
```

Comment:
what's the idea here? To expose this directory to the host system (or via shared volume) and then make that available to welder-web ?